cypress/e2e/04_set_api_key.cy.ts

```diff
@@ -0,0 +1,81 @@
+/// <reference types="cypress" />
+
+describe("studio", () => {
+  // This method performs setup before each test.
+  beforeEach(() => {
+    // Open Eyes to start visual testing.
+    // Each test should open its own Eyes for its own snapshots.
+    cy.eyesOpen({
+      // The name of the application under test.
+      // All tests for the same app should share the same app name.
+      // Set this name wisely: Applitools features rely on a shared app name across tests.
+      appName: "studio",
+
+      // The name of the test case for the given application.
+      // Additional unique characteristics of the test may also be specified as part of the test name,
+      // such as localization information ("Home Page - EN") or different user permissions ("Login by admin").
+      testName: Cypress.currentTest.title,
+    });
+  });
+
+  // The interactions use typical Cypress calls,
+  // but the verifications use one-line snapshot calls with Applitools Eyes.
+  // If the page ever changes, then Applitools will detect the changes and highlight them in the Eyes Test Manager.
+  // Traditional assertions that scrape the page for text values are not needed here.
+
+  it("should open flojoy studio's main page", () => {
+    cy.visit("/").wait(1000);
+
+    cy.eyesCheckWindow({
+      tag: "dark flow page",
+      target: "window",
+      fully: true,
+    });
```

Comment:
Pretty sure other tests already checks this window, make sure to remove all the duplicated tests.